### ISSUE TITLE
geforcenow-electron dropped support for deb packages

### ISSUE THREAD

**Issue by natanjunges** [wimpysworld/deb-get#295]
### Expected behavior

The package should be installed normally.

### Actual behavior

The developers announced in the latest release notes that the support for deb packages was dropped, so the release does not have a deb package any more:
```
  [+] Updating /var/cache/deb-get/geforcenow-electron.json
  [!] ERROR! Download URL for geforcenow-electron is empty. Skipping.
```

### Steps to reproduce

Run:
```bash
sudo deb-get install geforcenow-electron
```

#### /etc/os-release

```
PRETTY_NAME="Ubuntu 22.04 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

#### /etc/lsb-release

```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04 LTS"
```

#### deb-get version

```
0.3.0
```

### deb-get update

```
Hit:1 https://download.docker.com/linux/ubuntu jammy InRelease
Ign:2 http://packages.linuxmint.com una InRelease
Get:3 https://zotero.retorque.re/file/apt-package-archive ./ InRelease [2153 B]
Hit:4 http://archive.ubuntu.com/ubuntu jammy InRelease
Hit:5 http://packages.linuxmint.com una Release
Get:6 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [114 kB]
Get:9 http://archive.ubuntu.com/ubuntu jammy-backports InRelease [99.8 kB]
Hit:8 https://packagecloud.io/AtomEditor/atom/any any InRelease
Get:10 http://archive.ubuntu.com/ubuntu jammy-security InRelease [110 kB]
Fetched 326 kB in 2s (150 kB/s)
Reading package lists...
  [+] Updating /var/cache/deb-get/deb-get.json
```

**Comment by flexiondotorg** [comment]
Well, that is unfortunate. geforcenow-electron should be removed from `get-deb`.

I think adding a function that runs after any operations that warns of .debs installed via deb-get that are no longer being published upstream is a good idea.

This will not be the last time we will need to remove some thing from deb-get and we should be responsible in surfacing that to users.

**Comment by natanjunges** [comment]
We could use a convention that already exists in the code: changing the `deb_` prefix to either `disabled_` or `disable_` (or even something like `deprecated_`). Then we could simplify the functions (so that the check for `ACTION` not being `prettylist` is not necessary) and use them to detect the installed packages. We could even add a command to list them, and a section in the `README` with the results.

**Comment by flexiondotorg** [comment]
I like the idea of using `deprecated_`. I do think any package that has been deprecated needs to clearly warn the user, perhaps on `update` and `upgrade` actions.

**Comment by natanjunges** [comment]
Closed by #523.
